mineral-dart/core
# remove member from cache when ban

### What are the changes?
Removing the member from the server cache : 
`marshaller.cache.remove(memberCacheKey)`
And updating the server in the cache :
`marshaller.cache.put(serverCacheKey, rawServer)`

### How to test the changes?
Replace that :
```dart
await Future.wait([
      marshaller.cache.put(serverCacheKey, rawServer),
      marshaller.cache.put(channelCacheKey, rawChannel)
    ]);
```

By that :
```dart
marshaller.cache.has(memberCacheKey).then((value) => print(value));
    await Future.wait([
      marshaller.cache.remove(memberCacheKey),
      marshaller.cache.put(serverCacheKey, rawServer),
    ]).then((value) => marshaller.cache.has(memberCacheKey).then((value) => print(value)));
```

It should log true at the beginning when the member is not removed from the cache and false and the end when the member is removed.
